contrib/k8s/charts/open-service-broker-azure/values.yaml

```diff
@@ -49,6 +49,8 @@ tls:
 
 ## Basic auth credentials that can later be used to access this broker
 basicAuth:
+  ##K8's secret name that contains username and password for basic auth
```

Comment:
please also add "if specified, `username` and `password` will be ignored; if not specified, `username` and `password` should be provided for basic auth"